### PR TITLE
commit full setup before V2 campaign

### PR DESCRIPTION
**Pull request by scooperstein**
Updates to stay up-to-date with nanoAOD-tools. Anticipating a V2 launch of 2017 and possibly also 2016 with the puWeight and gen-event counting fixes. Also switched to deepCSV for 2017 but will keep track of CMVA-selected jets and kinematics as well.

### CODE REVIEW

File: crab/crab_data_2017.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	#if site=='T2_CH_CERN' :
+	#	config.Data.outLFNDirBase=  '/store/group/cmst3/group/nanoAOD/NanoTestProd006'
+	#else :
+   #		config.Data.outLFNDirBase = '/store/user/%s/NanoTestProd006/' % (getUsernameFromSiteDB())
+
+        config.Data.inputDataset = dataset
+	config.Data.unitsPerJob = 2000000
+	n+=1
+	nnn="%s"%n
+        config.General.requestName = "VHbbPostNano2017_V2_"+dataset.split('/')[1][:30]+dataset.split('/')[2][:30]+nnn
```
Comment (arizzi) [inline crab/crab_data_2017.py:53]: we need to add the trailing part of the dataset name also to the outputDatasetTag to separate the Run2017ABCDE...

File: crab/crab_script_2017.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#this is not mean to be run locally
```
Comment (arizzi) [inline crab/crab_script_2017.sh:1]: why do we need a different one for 2017?

File: crab/crab_script_2017_data.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#this is not mean to be run locally
```
Comment (arizzi) [inline crab/crab_script_2017_data.sh:1]: why do we need a different one for data?

File: crab/crab_data_2017.py (hunk 1 above)
Comment (scooperstein) [reply crab/crab_data_2017.py:53]: how about we but just below this line something like
config.General.outputDatasetTag = dataset.split('/')[2][:30]+nnn
?

**Comment by scooperstein** [comment]
hi @arizzi , I just added commit in response to your feedback. Let me know what you think.

**Pull request merged**